grammar.js

```diff
@@ -1047,9 +1047,9 @@ module.exports = grammar({
     rational: $ => seq($._int_or_float, token.immediate('r')),
     super: $ => 'super',
     self: $ => 'self',
-    true: $ => token(choice('true', 'TRUE')),
-    false: $ => token(choice('false', 'FALSE')),
-    nil: $ => token(choice('nil', 'NIL')),
```

Comment:
I was about to say the upper-case variants were still valid as of 2.7 (but not in 3.0), but it looks like they were only ever [defined as constants](https://github.com/ruby/ruby/commit/62554ca97812b454e78a0a9daf6e962ff7a2f589), and never parser keywords, so 👍 to this change.